packages/provider/src/index.ts

```diff
@@ -60,12 +67,12 @@ export function initInpageProvider({ onDispatch, env }: initInpageProviderArgs)
           const response =
             typeof event.data === 'object' ? event.data : JSON.parse(event.data as any);
           if (response.id !== id) return;
-          window.removeEventListener('message', handleMessage);
+          removeEventListener(handleMessage);
           // eslint-disable-next-line @typescript-eslint/prefer-promise-reject-errors
           if ('error' in response) return reject(response);
```

Comment:
is `response.error` a string? Wouldn't it better to `reject(new Error(error))` instead of working around the eslint rule?

Reply:
Missed that this isn't part of the PR, but still hopefully an easy fix.